src/phoenix/server/main.py

```diff
@@ -187,7 +187,9 @@ def _load_items(
     traces = Traces()
     if trace_dataset_name is not None:
         fixture_spans = list(
-            encode(json_string_to_span(json_span))
+            # Apply `encode` here because legacy jsonl files contains UUIDs as strings.
+            # `encode` removes the hyphens in the UUIDs.
```

Comment:
I think in general we don't really have JSONL now other than fixtures right?

Reply:
that's right